README.md

```diff
@@ -7,37 +11,58 @@ A client/wrapper for foursquare's sparse geocoding / reverse geocoding server Tw
 
 Add this line to your application's Gemfile:
 
-    gem 'twofishes'
+```ruby
+gem 'twofishes'
+```
 
 And then execute:
 
-    $ bundle
+```sh
+$ bundle
+```
 
 Or install it yourself as:
 
-    $ gem install twofishes
+```sh
+$ gem install twofishes
+```
 
 ## Configuration
 
-    Twofishes.configure do |config|
-      config.base_url = 'http://mytwofishes.com:8081'
-      config.timeout = 3 # in seconds
-    end
+```ruby
+Twofishes.configure do |config|
+  config.host = '127.0.0.1' # server address
+  config.port = 8080 # thrift port
+  config.timeout = 3 # timeout in seconds
+  config.retries = 2 # how many times to retry a request
+end
+```
+
+Timeouts are enforced per-try, so if you have a timeout of n and do m retries, the total time it could take is n*m.
 
 ## Usage
 
 Use the following methods to geocode / reverse geocode.
 
-    Twofishes::Client.reverse_geocode([0, 0])
-    Twofishes::Client.geocode('zurich')
-
-If you want more control over the params, you can also use the ```call_api``` method directly.
+```ruby
+Twofishes::Client.geocode('Ljubljana')
+Twofishes::Client.geocode('Zurich', [ResponseIncludes::PARENTS])
```

Comment:
change documentation for `[ResponseIncludes::PARENTS]` because you will change it to options hash syntax